crow/orders/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db    import models
+
+from users.models import TimeStampedModel
+
+class OrderStatus(models.Model):
+    status = models.CharField(max_length=40)
+
+    class Meta:
+        db_table = "orderstatus"
+
+class PaymentMethod(models.Model):
+    payment = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = "paymentmethod"
+
+class DeliveryAddress(models.Model):
+    deliveryaddress = models.CharField(max_length=200)
```

Comment:
```suggestion
    delivery_address = models.CharField(max_length=200)
```
해당 테이블엔 user 정보가 없어도 되나요?

Reply:
아뇨 수정했습니다!